data/blog/astral-projection-release-recap-oct-2023.mdx

```diff
@@ -13,12 +14,38 @@ We’re always shipping new features and fixes at Tailscale — so much so that
 
 This month’s updates are all written by me, Parker, but we’re keeping the blurbs signed and you might get more perspectives from different teams here in coming months. And if you’d like to share how you’re using a newly released Tailscale feature, get in touch at devrel\@tailscale.com, on [Twitter](https://twitter.com/tailscale), or on the [fediverse](https://hachyderm.io/@tailscale), and we may include your story in a future post.
 
+This post is also available as YouTube Short, if you'd prefer to watch it in video form:
```

Comment:
"as a YouTube short"?

Reply:
ha, I had that first and changed it. "YouTube Shorts" is the formal feature name so I think we cap following their style